ts/image-occlusion/Toolbar.svelte

```diff
@@ -189,4 +239,31 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         width: 0.1em !important;
         height: 0.1em !important;
     }
+
+    .tool-bar-container {
+        position: fixed;
+        top: 42px;
+        left: 2px;
+        height: 100%;
+        border-right: 1px solid var(--border);
+        overflow-y: auto;
+        width: 32px;
+        z-index: 99;
+        background: var(--canvas-elevated);
+        padding-bottom: 100px;
+    }
+
+    :global(.tool-icon-button) {
+        border: unset;
+        display: block;
+        width: 32px;
+        height: 32px;
+        margin: unset;
+        padding: 6px !important;
+    }
+
+    :global(.active-tool) {
+        color: white !important;
+        background: var(--button-primary-bg) !important;
+    }
```

Comment:
Do any of these need to be global?

Reply:
If class it on HTML element then global cannot be used but when class need to add to component then following issues is showing 
`Unused CSS selector ".active-tool"svelte(css-unused-selector)`